pkg/plugins/policies/meshhttproute/xds/configurer.go

```diff
@@ -39,11 +39,16 @@ func (c RoutesConfigurer) Configure(virtualHost *envoy_route.VirtualHost) error
 		// a prefix match along to the filters because it's no longer
 		// possible to know for sure with just an envoy_route.Route
 		for _, filter := range c.Filters {
-			rb.
-				Configure(filters.NewRequestHeaderModifier(filter.RequestHeaderModifier)).
-				Configure(filters.NewResponseHeaderModifier(filter.ResponseHeaderModifier)).
-				Configure(filters.NewRequestRedirect(filter.RequestRedirect, match.prefixMatch)).
-				Configure(filters.NewURLRewrite(filter.URLRewrite, match.prefixMatch))
+			switch filter.Type {
```

Comment:
Not a blocker but I think better would be to avoid inlining like this because this will end up being the exact same code duplicated anywhere we use a `meshhttproute.Filter`.